lib/create-configs/__tests__/create-configs-test.js

```diff
@@ -0,0 +1,221 @@
+const assert = require('assert')
+const { join } = require('path')
+
+const createConfigs = require('../')
+
+it(`
+  when no options are pased,
+  a serverConfig and clientConfig is returned
+  containing a default output.path
+`, () => {
+  const { serverConfig, clientConfig } = createConfigs()
+  const { output: { path: clientOutputPath } } = clientConfig
+  const { output: { path: serverOutputPath } } = serverConfig
```

Comment:
A good idea. A lot of the awkwardness here is from translating this test file from testing `brb` to just the config helper.